src/marqo/tensor_search/tensor_search.py

```diff
@@ -2648,3 +2657,33 @@ def delete_documents(config: Config, index_name: str, doc_ids: List[str]):
             document_ids=doc_ids,
         )
     )
+
+def normalize_vector(embeddings: Union[List[List[float]], ndarray]) -> List[List[float]]:
```

Comment:
Was there no existing normalization code? We already normalize for non-custom vectors

Reply:
There was no existing function that could be reused to normalize vector, since normalization is being done in 3 places (add docs - structured, add docs - unstructured & query docs) I created a function for reusability purposes.